lib/dogapi/v1/synthetics.rb

```diff
@@ -0,0 +1,80 @@
+require 'dogapi'
+
+module Dogapi
+  class V1 # for namespacing
+    # SyntheticsService is the class responsible for dealing with the synthetics
+    class SyntheticsService < Dogapi::APIService
+
+      API_VERSION = 'v1'
+
+      # Create a synthetics test: POST /v1/synthetics/tests/
+      def create_test(type, config, options = {})
+        body = {
+          'type' => type,
+          'config' => config
+        }.merge(options)
+
+        request(Net::HTTP::Post, "/api/#{API_VERSION}/synthetics/tests", nil, body, true)
+      end
+
+      # Edit a synthetics test: PUT /v1/synthetics/tests/<SYNTHETICS_TEST_PUBLIC_ID>
+      def edit_test(test_id, type, config, options = {})
```

Comment:
`update` instead of `edit`?

Reply:
The API docs use `edit` https://docs.datadoghq.com/api/?lang=bash#edit-a-test so I went with that (although the docs also mention `update` in the description).